src/html/scan.js

```diff
@@ -470,6 +470,8 @@ function submitOptions(e) {
   const formObject = Object.fromEntries(new FormData(formElem));
   // Add in all the unchecked checkboxes which will be absent from a FormData object
   formElem.querySelectorAll('input[type=checkbox]:not(:checked)').forEach((k) => formObject[k.name] = false);
+  // Force customised to true as this is now customising it
+  formObject['customised'] = true;
```

Comment:
I think it would be better to set customised true just in the backend code?

Reply:
Sadly, the backend doesn't refresh the options json when you POST it, so the values returned by the call are still the old values (including customized) and can't be used. The options JSON string is only refreshed on boot. Recreating the string in the middle of the request is somewhat RAM intensive and I avoided bringing that up because our ESP wifi support is already so m-f'ing unreliable that I didn't want to chance breaking it for something the user probably wouldn't notice unless they refresh the page before rebooting.